codegen/example/xata.js

```diff
@@ -9,7 +9,7 @@ import { BaseClient, RestRespositoryFactory } from "../../client/src";
  * @property {() => Promise<void>} delete
  * @property {string=} name
  * @property {string[]=} labels
- * @property {User=} owner
+ * @property {UserRecord=} owner
```

Comment:
Not for this PR, but I wonder if we could generate a .d.ts when generating JS code and importing these types instead of using `@property` annotations. This is what we talked a few weeks ago and forgot to open an issue. I'll create one.